### ISSUE TITLE
Add a more explicit Github link to the website

### ISSUE THREAD

**Issue by rdimitrov** [theupdateframework/theupdateframework.io#28]
It would be useful to have a more explicit link to Github (to the specification repository) on the main panel in the form of a dedicated tab or use Github's logo.

**Comment by h4l0gen** [comment]
Hii @rdimitrov can you explain me suggested implementation in more detail, we have already `Getting started ->Specification(latest)` section.

**Comment by rdimitrov** [comment]
hey, @h4l0gen, 

I think in the issue what I meant was there's no easy way to navigate from https://theupdateframework.io/ to the GitHub repository hosting the specification - https://github.com/theupdateframework/specification

Note: the link I was able to find redirects to the python-tuf implementation.

**Comment by h4l0gen** [comment]
Yes @rdimitrov there is only link to [Specification(index)](https://theupdateframework.github.io/specification/latest/). As per our [discussion ](https://github.com/theupdateframework/theupdateframework.io/pull/29)I've started work on creating contributing page in [this issue](https://github.com/theupdateframework/theupdateframework.io/issues/49). IMO we should add all repository links there within `your first contributing` section. WYT? Should we add `contributing` and `your first contribution` section inside `Getting started` or create a new page `Contributing` like this
![image](https://github.com/theupdateframework/theupdateframework.io/assets/117383853/bd4dd0a0-9062-44bc-89ab-d632bf0225e0)
